src/sampleGenerator/security.js

```diff
@@ -0,0 +1,26 @@
+require("dotenv").config();
+const fs = require("fs");
+const path = require("path");
+const sampleDataPath = path.resolve(".", "sampleData.json");
+
+const loadSampleData = new Promise((resolve, reject) => {
+  fs.readFile(sampleDataPath, (err, data) => {
+    if (err) {
+      reject(err);
+    } else {
+      resolve(JSON.parse(data));
+    }
+  });
+});
```

Comment:
```suggestion
const loadSampleData = async () => {
  const data = fs.readFileSync(sampleDataPath);
  return JSON.parse(data));
});
```

Reply:
오 샘플 데이터 읽는 부분은 작성해주신 것처럼 비동기처리가 없어도 문제 없어보여요. 이렇게 되면 비동기 로직이 없어지니까 함수 선언에서 async를 빼야 합니다!

Reply:
생각해보니깐 그럼 굳이 함수일 필요도 없는듯요?? ㅋㅋㅋ

```suggestion
const sampleData = JSON.parse(fs.readFileSync(sampleDataPath));
```

Reply:
오 그러네요!! 그리고 loadSampleData를 호출하는 코드도 고쳐야 해요

Reply:
반영 완료했습니다! 좋은 의견 감사해요!